systems/registry/api-gateway/pkg/client/registry.go

```diff
@@ -252,3 +252,42 @@ func (r *Registry) GetSites(netID string) (*netpb.GetSitesByNetworkResponse, err
 
 	return res, nil
 }
+
```

Comment:
In all these 3 functions  `AddInviation`, `GetInvitation`  and `UpdateInvation`,   the following 
```go
if err != nil {
		return nil, err
	}

	return res, nil
```
is not necessary.  You can return directly the result of the RPC call, like 

`return r.orgClient.AddInvitation(ctx, &orgpb.AddInvitationRequest{Email: email,Org: org,Role: orgpb.RoleType(orgpb.RoleType_value[role]),Name: name})`

`return r.orgClient.GetInvitation(ctx, &orgpb.GetInvitationRequest{Id: invitationId,})`

`return r.orgClient.UpdateInvitation(ctx, &orgpb.UpdateInvitationRequest{Id: invitationId, Status: orgpb.InvitationStatus(orgpb.InvitationStatus_value[status])})`